SeaPublicWebsite/Views/Shared/_Recommendation.cshtml

```diff
@@ -1,131 +1,69 @@
 ﻿@using GovUkDesignSystem
 @using GovUkDesignSystem.GovUkDesignSystemComponents
-@using SeaPublicWebsite.Controllers
-@using SeaPublicWebsite.Models.EnergyEfficiency
-@model SeaPublicWebsite.Models.EnergyEfficiency.RecommendationViewModel;
+@model SeaPublicWebsite.Models.EnergyEfficiency.SingleRecommendationDetailsViewModel;
 
-@{
-    ViewBag.Title = "Recommendation " + (Model.RecommendationIndex + 1) + " of " + Model.PropertyRecommendations.Count;
```

Comment:
Does this page not get a title any more? Or is it being set elsewhere?

Reply:
There has been a bit of a reshuffle so that the _Recommendation shared layout only covers rendering the core chunk of the recommendations page that is shared across the pdf and the html page.

YourRecommendation.cshtml contains the rest of the html page, including the title and the layout.

The page that will become the pdf doesn't have a title set as I didn't think it was necessary though happy to set one if it does anything.

Reply:
Awesome, thanks for explaining. I don't expect the pdf needs one so fine to leave as-is